TPI/app/src/main/java/biz/stratadigm/tpi/entity/vo/VenueVO.java

```diff
@@ -0,0 +1,71 @@
+package biz.stratadigm.tpi.entity.vo;
+
+public class VenueVO {
+    private Long id;
+    private String name;
+    private Integer thalis;
+    private Loc loc;
+
+    public VenueVO(Long id, String name, Integer thalis, Float lat, Float lng) {
+        this.id = id;
+        this.name = name;
+        this.thalis = thalis;
+        this.loc = new VenueVO.Loc();
+        this.loc.setLat(lat);
+        this.loc.setLng (lng);
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Integer getThalis() {
+        return thalis;
+    }
+
+    public void setThalis(Integer thalis) {
+        this.thalis = thalis;
+    }
+
+    public Loc getLoc() {
+        return loc;
+    }
+ 
+    public void setLoc(Loc loc) {
+        this.loc = loc;
+    }
+
+    public static class Loc {
```

Comment:
Please, don't shorten entity names too much. This name makes me confuesed. Can you please rename it to Location?